### ISSUE TITLE
Demonstrate Population in the current examples

### ISSUE THREAD

**Issue by CrashyBang** [pubkey/rxdb#72]
## Case

Request.

## Issue

The current examples do not demonstrate [Population](https://github.com/pubkey/rxdb/blob/master/docs/Population.md) also the current documentation a little hard to follow (I am working on a PR but I am need to get my head around it more first) hence why an example may be productive.

**Comment by kethan** [comment]
What about array of reference? And nested population ?

**Comment by pubkey** [comment]
NOTE: I'm currently working on this. Fix comes soon
